tests/test_runs/test_run_functions.py

```diff
@@ -361,17 +412,75 @@ def determine_grid_size(param_grid):
             self.assertTrue(check_res)
 
         # todo: check if runtime is present
-        self._check_fold_evaluations(run.fold_evaluations, 1, num_folds)
+        self._check_fold_evaluations(run.fold_evaluations, 1, num_folds,
+                                     task_type="Supervised Classification")
+        pass
+
+    def _run_and_upload_regression(self, clf, rsv):
+        def determine_grid_size(param_grid):
+            if isinstance(param_grid, dict):
+                grid_iterations = 1
+                for param in param_grid:
+                    grid_iterations *= len(param_grid[param])
+                return grid_iterations
+            elif isinstance(param_grid, list):
+                grid_iterations = 0
+                for sub_grid in param_grid:
+                    grid_iterations += determine_grid_size(sub_grid)
+                return grid_iterations
+            else:
+                raise TypeError('Param Grid should be of type list'
+                                '(GridSearch only) or dict')
+
+        task_id = 738  # quake dataset
+        num_test_instances = 718  # 33% holdout task
+        num_folds = 1  # because of holdout
+        num_iterations = 5  # for base search classifiers
+
+        run = self._perform_run(task_id, num_test_instances, clf,
+                                random_state_value=rsv)
+
+        # obtain accuracy scores using get_metric_score:
```

Comment:
Could you please rename all references to classification (such as accuracy) to reference regression?